tsl/test/expected/compression_bgw.out

```diff
@@ -205,3 +205,66 @@ ERROR:  permission denied to start compress_chunks background process as role "n
 \set ON_ERROR_STOP 1
 RESET ROLE;
 REVOKE NOLOGIN_ROLE FROM :ROLE_DEFAULT_PERM_USER;
+\c :TEST_DBNAME :ROLE_SUPERUSER
```

Comment:
We shouldn't run tests as super user.

Reply:
The user is not relevant for this test since we are not testing permissions nor the tests are dependent on permissions.

Reply:
Not running as super-user, unless necessary, has been a rule of thumb that has served us well and I think we should continue to avoid it.

There are several reasons for this. 

1. Running as super-user often hides secondary/dependent issues that we won't discover if we continue doing it
2. Someone that later adds to this test might not notice that the user is not super-user and then unintentionally run the additional tests as super-user.